triphouse/index.html

```diff
@@ -6,6 +6,54 @@
     <link href="style.css" rel="stylesheet">
 </head>
 <body>
```

Comment:
set margin: 0 to body

Reply:
add margin. please see the form. something doesn't work when flex is turned on